pkg/Apps/Graph/Library/GraphBuilderNode.js

```diff
@@ -13,9 +13,9 @@ const globalStores = [
   'nodeId',
 ];
 
-export const RecipeBuilderRecipe = {
+export const GraphBuilderNode = {
```

Comment:
Can't be a node unless it's has a $meta and an id. I think suspect these objects in Graph/Library simply still Recipes.

Actual Nodes for these things are in (main) Library/Node.

Reply:
i reverted all the recipe->node renaming changes... 
i'm not sure we should keep the word "recipe" in our glossary, there are probably "node types" now... let's leave it as is for now and discuss?

Reply:
Sgtm.